conf/uppmax.config

```diff
@@ -10,31 +10,34 @@ vim: syntax=groovy
  */
 
 process {
-  executor = 'local'
-  cpus = { 1 * task.attempt }
-  memory = { 8.GB * task.attempt }
-  time = { 2.h * task.attempt }
+  executor = 'slurm'
   clusterOptions = { "-A $params.project ${params.clusterOptions ?: ''}" }
   errorStrategy = { task.exitStatus == 143 ? 'retry' : 'finish' }
   maxRetries = 3
   maxErrors = '-1'
 
   $fusioncatcher{
     module = ['bioinfo-tools', 'FusionCatcher/0.99.5a']
-    cpus = { 16 * task.attempt }
-    memory = { 32.GB * task.attempt }
+    time = { 72.h * task.attempt }
+    //cpus = { 8 * task.attempt }
+    cpus = { 14 }
   }
   $star_fusion{
     module = ['bioinfo-tools', 'star-fusion']
-    cpus = { 4 * task.attempt }
-    memory = { 32.GB * task.attempt }
+    time = { 30.h * task.attempt }
+    cpus = { 8 * task.attempt }
   }
+  $fusioninspector{
+    module = ['bioinfo-tools', 'star-fusion', 'trinity/2014-07-17', 'samtools/1.5', 'htslib/1.5', 'perl_modules']
```

Comment:
Can drop this now that we have singularity?

See examples here for how to build the config for singularity / docker:

https://github.com/SciLifeLab/NGI-RNAseq/blob/master/nextflow.config#L13-L16
https://github.com/SciLifeLab/NGI-RNAseq/blob/master/conf/uppmax.config#L19
https://github.com/SciLifeLab/NGI-RNAseq/blob/master/conf/uppmax.config#L19
https://github.com/SciLifeLab/NGI-RNAseq/blob/master/conf/docker.config#L14-L20